.github/workflows/pr_pre-commit.yml

```diff
@@ -18,10 +19,22 @@ jobs:
       - uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
         with:
           python-version: '3.12'
-      - uses: dtolnay/rust-toolchain@master
```

Comment:
Is this not required? I thought it was, for clippy, etc...

Reply:
Yeah, it is required. I accidentally removed it.

Reply:
Re-added in 75fed3e